docs/src/joint/api/elementTools/Button.html

```diff
@@ -31,6 +31,11 @@
         <td><i><a href="#dia.Cell.markup.json">JSONMarkup</a></i></td>
         <td>The markup of the button, provided in the <a href="#dia.Cell.markup.json">JointJS JSON format</a>. Default is <code>undefined</code> (no content).</td>
     </tr>
+    <tr>
+        <th>scale</th>
+        <td><i>number</i></td>
+        <td>Scale the button up or down on the 2D plane. Default is <code>1</code>.</td>
```

Comment:
I would change this sentence to the following in all examples:

"Scale the (tool) up or down on **a** 2D plane. **The** default is 1."